src/OrchardCore/OrchardCore.Infrastructure/Email/OrchardCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,24 @@
+using OrchardCore.Email;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    /// <summary>
+    /// Provides an extension method for <see cref="OrchardCoreBuilder"/>.
+    /// </summary>
+    public static partial class OrchardCoreBuilderExtensions
+    {
+        /// <summary>
+        /// Adds e-mail address validator service.
+        /// </summary>
+        /// <param name="builder">The <see cref="OrchardCoreBuilder"/>.</param>
+        public static OrchardCoreBuilder AddEmailAddressValidator(this OrchardCoreBuilder builder)
+        {
+            builder.ConfigureServices(services =>
+            {
+                services.AddTransient<IEmailAddressValidator, EmailAddressValidator>();
+            });
```

Comment:
`Scoped` I think not transient

Reply:
Transient is okay as it has no state and it allows to be injected by a singleton service. It could also have been a singleton, and if we are sure that we always use this implementation, maybe a static component that don't need to be injected.

Reply:
Transient and Singleton are good, but from DI as we need devs to be able to customize the logic.